.travis.yml

```diff
@@ -6,10 +6,18 @@ sudo: false
 cache:
   directories:
   - $HOME/.m2
+  - downloads
 
 jdk:
+  - oraclejdk9
   - oraclejdk8
   - openjdk7
 
 matrix:
   fast_finish: true
+
+before_install:
+ - mkdir -p downloads
+ - if [[ ! -f downloads/maven.tar.gz ]]; then curl -o downloads/maven.tar.gz http://mirror.ox.ac.uk/sites/rsync.apache.org/maven/maven-3/3.5.0/binaries/apache-maven-3.5.0-bin.tar.gz; fi
+ - tar xf downloads/maven.tar.gz
+ - export PATH=$(pwd)/apache-maven-3.5.0/bin:$PATH
```

Comment:
From https://travis-ci.org/ome/ome-common-java/jobs/279470866

```
mvn version

Apache Maven 3.5.0 (ff8f5e7444045639af65f6095c62210b5713f426; 2017-04-03T19:39:06Z)
Maven home: /usr/local/maven-3.5.0
```

so the latest Travis images ship with Maven 3.5.0 by default. What is the rationale behind this `before_install`?

Reply:
This work dates from June, and the maven provided by travis was an old version which didn't work with java 9 at all.  If the images are updated, we can drop this entirely.